app/models/position_opening.rb

```diff
@@ -141,28 +231,40 @@ def search_for(options = {})
     end
 
     def delete_search_index
-      search_index.delete
+      client.indices.delete index: INDEX_NAME rescue nil
     end
 
-    def search_index
-      Tire.index(index_name)
+    def search_index_exists?
+      client.indices.exists? index: INDEX_NAME
     end
 
     def import(position_openings)
-      Tire.index index_name do
-        import position_openings do |docs|
-          docs.each do |doc|
-            doc[:id] = "#{doc[:source]}:#{doc[:external_id]}"
-            doc[:locations].each do |loc|
-              normalized_city = loc[:city].sub(' Metro Area', '').sub(/, .*$/, '')
-              lat_lon_hash = Geoname.geocode(location: normalized_city, state: loc[:state])
-              loc[:geo] = lat_lon_hash if lat_lon_hash.present?
-            end if doc[:locations].present?
+      position_openings.each do |opening|
+        data = opening.except(:_timestamp, :_ttl).each_with_object({}) do |(key, value), data|
```

Comment:
If `_timestamp` and `_ttl` are no longer special fields in Elasticsearch (they aren't), then I'd like the importer to no longer expect them to have the leading underscores in the field names. It's just unnecessarily mysterious.